skyportal/models/schema.py

```diff
@@ -136,10 +136,11 @@ def add_schema(schema_class_name, exclude=[], add_to_model=False):
                     {
                         'model': class_,
                         'sqla_session': _DBSession,
+                        'load_instance': True,
                         'ordered': True,
                         'exclude': [],
                         'include_fk': True,
-                        'include_relationships': False,
+                        'include_relationships': True,
```

Comment:
What do this changes actually do?

Reply:
Not quite sure, but it was one of the required changes as specific in stack overflow to make it happen.

Reply:
Let's make sure the test failures are unrelated then merge.